### DISCUSSION TITLE
Is it possible to use a different ImGui version?

### DISCUSSION THREAD

**Discussion by mnesarco** [DiligentGraphics/DiligentEngine#148] [General]
Hi Friends,

I am just learning DiligentEngine, it looks very good. This is a noob question: I want to use the docking branch of ImGui, is there any option to select the imgui version used? someting like `DILIGENT_IMGUI_DIR=<whatever>` ? or do I have to override the files in `DiligentTools/ThirdParty/imgui` ?

Thanks.

**Comment by TheMostDiligent** [comment]
Hi!
Imgui is included as a submodule. You can make this submodule point to any commit

**Reply by mnesarco** [reply]
Thank you.
